wallet/src/main/scala/org/bitcoins/wallet/config/WalletAppConfig.scala

```diff
@@ -97,6 +98,12 @@ case class WalletAppConfig(
     confs
   }
 
+  lazy val aesPassword: AesPassword = {
```

Comment:
So I think we should default to not encrypting the data on disk if the password is not specified rather than using `BIP39KeyManager.badPassphrase`. 

We will need some sort of shim for old wallets that do use `BIP39KeyManager.badPassphrase` to encrypt their wallet, but I don't think we should continue this?